reactrestaurant/src/App.js

```diff
@@ -4,26 +4,34 @@ import './App.css';
 import Navbar from './components/navbar';
 import Footer from './components/footer';
 import Page from './components/page';
-
+import Accordian from './components/accordian';
+import './themestr.css';
+import NavTabs from './components/navtabs';
 /*
 navbar top
 carousel
   restuarant name
 */
-function App() {
-  return (
-    <>
-      <Navbar /> 
-      <Page />
-      <Footer />
-    </>
-    /*
-    state {0:home, 1:menu}
-    state == 0 ? jumbotron+carousel : menu, title, col, accordians, subtitle, dishes
-    */
+class App extends React.Component {
```

Comment:
Hi Michael, I didn't get a chance to look this over before it got merged, so I'll try to offer some thoughts, but feel free to come find me if you have extra questions, or if I can be of help.